src/components/Profile.jsx

```diff
@@ -17,4 +18,12 @@ class Profile extends React.Component {
 	}
 }
 
+Profile.propTypes = {
+	user: PropTypes.shape({
+		images: PropTypes.array,
+		firstName: PropTypes.string,
+		lastName: PropTypes.string,
+	}),
```

Comment:
We access `user`'s properties without checking for it, so it should be required.